### ISSUE TITLE
Why did you loss 0.01 after 1 epoch when you started training from zero？

### ISSUE THREAD

**Issue by xiaoxin05** [wbenbihi/hourglasstensorlfow#68]
hi，First of all, thank you for sharing
1）I see that I'm using the cross-entropy loss function here？
2）The training data was 20948，but  An epoch appears later，loss is 0.01？
